dags/common/request.py

```diff
@@ -0,0 +1,45 @@
+from requests.exceptions import RequestException
+
+class Request:
+    def __init__(
+        self,
+        base_url: str = 'localhost',
+        port: int = 8080,
+        parameters: dict = {},
+        subdir: str ='',
+        headers: dict = {}
+    ):
+        self.base_url = base_url 
+        self.parameters = parameters
+        self.headers = headers
+        self.subdir = subdir
+
+    def __repr__(self) -> str:
+        return(f"< RestApi base_url={self.base_url}, parameters={self.parameters}, header={self.headers} >")
+        
+    def get_response(self):
+        from furl import furl
+        import requests
+
+        url_base_obj = furl(self.base_url)
+        if bool(self.subdir):
+            url_base_obj = url_base_obj.path(self.subdir)
+        if bool(self.parameters):
+            url_base_obj = url_base_obj.add(self.parameters)
+        if bool(self.headers):
+            response = requests.get(url_base_obj.url, headers=self.headers)
+            return response
+        response=requests.get(url_base_obj.url)
```

Comment:
you don't have to do the `if`s since the library will take care it if the value is `None`